pkg/ostree/client.js

```diff
@@ -388,6 +388,24 @@ function RPMOSTreeDBusClient() {
         return proxy;
     };
 
+    self.signature_obj = function(signature) {
+        if (!signature.v)
+            return;
+
+        var by = signature.v[11];
```

Comment:
We should have a multiline comment here, either with a paste of the variant signature, but preferrably to the place where it's defined ... if possible a link to documentation.